File: Medicanna/server/src/main/java/ehealth/data_objects/BaseResponse.java
package ehealth.data_objects;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

import java.util.UUID;


@EqualsAndHashCode
@AllArgsConstructor
@Getter
@Setter
@NoArgsConstructor
@ToString
public class BaseResponse {
    @JsonProperty("request_id")
    UUID requestId;

    @JsonProperty("status")
    String status;

    @JsonProperty("body")
    String body;

    public BaseResponse(String status) {
        this.status = status;
    }
}

File: Medicanna/client/CannaBe/CannaBe/DataObjects/Http/Response/BaseResponse.cs
using Newtonsoft.Json;
using System.Net.Http;

namespace CannaBe
{
    class BaseResponse : IJsonResponse
    {
        [JsonProperty("request_id")]
        public string RequestId { get; set; }

        [JsonProperty("status")]
        public string Status { get; set; }

        [JsonProperty("body")]
        public string Body { get; set; }

        [JsonConstructor]
        public BaseResponse(string requestId, string status, string body)
        {
            RequestId = requestId;
            Status = status;
            Body = body;
        }
    }
}
